content/en/containers/amazon_ecs/apm.md

```diff
@@ -97,28 +98,34 @@ The [Amazon's ECS container metadata file][1] allows discovery of the private IP
 {{< code-block lang="curl" >}}
 cat $ECS_CONTAINER_METADATA_FILE | jq -r .HostPrivateIPv4Address
 {{< /code-block >}}
-
     
 [1]: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/container-metadata.html#metadata-file-format
 {{% /tab %}}
 {{< /tabs >}}
 
-Set the result as your Trace Agent hostname environment variable for each application container shipping to APM:
+Proivde the result of this request to the Tracer through the environment variable `DD_AGENT_HOST` for each application container sending APM traces.
 
-{{< code-block lang="curl" >}}
-os.environ['DD_AGENT_HOST'] = <EC2_PRIVATE_IP>
-{{< /code-block >}}
-    
-    
+### Set Tracer hostname
```

Comment:
I don't think `Set Tracer hostname` is quite right for this because the hostname means something different and appears on the Infrastructure List and on the host.

What do you think of this heading?
```git suggest
Send Traces To the Datadog Agent Container
```

Reply:
The parent header of this is "Configure your application container to submit traces to Datadog Agent" so wanted to avoid those headers looking too much alike. The doc had mentioned `hostname` quite a bit which is technically correct, but ya could cause confusion here. 

I can change to "Configure the Tracer endpoint" and toy around with the wording there

Reply:
What about `Configure the Trace Agent URL` or `Configure the Trace Agent Endpoint` or something like that?

`Tracer endpoint` will cause confusion since the endpoint is to submit traces to the Agent, not the tracer.

Reply:
I'm good with `Configure the Trace Agent Endpoint` will update